zookeeper-metrics-providers/zookeeper-prometheus-metrics/src/main/java/org/apache/zookeeper/metrics/prometheus/PrometheusMetricsProvider.java

```diff
@@ -77,12 +73,14 @@ public class PrometheusMetricsProvider implements MetricsProvider {
      * Default value is 1.
      * If the number is less than 1, the main thread will be used.
      */
+    @Deprecated
     static final String NUM_WORKER_THREADS = "numWorkerThreads";
 
     /**
      * The max queue size for Prometheus summary metrics reporting task.
      * Default value is 1000000.
      */
+    @Deprecated
```

Comment:
Please add a comment (`@deprecated`) for why it's deprecated and/or what should be used instead.

Reply:
Please update the admin doc for the deprecation.

Reply:
updated